.circleci/config.yml

```diff
@@ -1354,6 +1354,44 @@ workflows:
       - aws-summary:
           context: Slack
 
+  nightly-comprehensive-functional-regression:
+    triggers:
+      - schedule:
+          cron: "15 4 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - fast-build-artifact:
+          context: Slack
+          workflow-name: freeze restart
```

Comment:
copy-paste issue I think. `workflow-name` should be `nightly-comprehensive-functional-regression` I guess

Reply:
Yeah, fixed.